CMake/vtkSlicerVersionConfigureMinimal.h.in

```diff
@@ -41,6 +41,31 @@
 /// Patch version number of this project source and binaries.
 #define Slicer_VERSION_PATCH @Slicer_VERSION_PATCH@
 
+/// \def Slicer_VERSION_NUMBER_COMPUTE(major, minor, patch)
+/// \brief Macro that computes Slicer_VERSION_NUMBER from major, minor, and patch numbers.
+/// This can be used for comparing Slicer version in #if/#else directives.
+/// This is useful when implementing Slicer extensions that need to be compatible
+/// with multiple Slicer core versions.
+///
+/// Example:
+///
+/// #if Slicer_VERSION_NUMBER >= Slicer_VERSION_NUMBER_COMPUTE(5, 9, 0)
+///   ... code compatible with Slicer-5.9.0 and newer
+/// #else
+///   ... code compatible with older Slicer versions
+/// #endif
+///
+/// Available since Slicer-5.7.0.
+/// Implementation is based on VTK_VERSION_CHECK (inspired by QT_VERSION_CHECK).
```

Comment:
I agree that `Slicer_VERSION_NUMBER_COMPUTE` is more explicit about the intent.

The original macro name seems to pre-date Qt4, it was already published 15 years ago[^1] in the commit titled `Long live Qt for S60!`. 

It then had to be maintained for backward compatibility

![image](https://github.com/user-attachments/assets/ab701a5b-5a2f-4377-921f-d11195541c19)

[^1]: https://github.com/qt/qt/blame/4.8/src/corelib/global/qglobal.h#L52-L66